README.md

```diff
@@ -3,209 +3,89 @@ Rig
-
-    py.test --spinnaker SPINN_HOSTNAME 8 8 --spinn5
-
-Other tests require a connected BMP (Board Management Processor, part of
-SpiNN-5 boards). To run these use:
-
-    py.test --bmp BMP_HOSTNAME
-
-When a BMP is connected, the test suite will attempt to power-cycle the
-attached board. When a SpiNNaker system is connected, the test suite will
-attempt to boot the system.
-
-To skip the power-cycling and booting tests, simply add the `--no-boot`
-argument.
-
-### Running tests against remote hardware
-
-Booting a SpiNNaker board requires the (reliable) sending of packets to UDP
-port 54321 which is frequently blocked by ISPs and is not reliable (since UDP
-gives no guarantees, especially on the open internet). As a result, a proxy
-server must be used to communicate with the board. A utility such as
-[`spinnaker_proxy`](https://github.com/project-rig/spinnaker_proxy) can be used
-alongside the test suite as follows:
-
-    # On the test machine
-    spinnaker_proxy.py -ctq PROXY_SERVER_HOSTNAME &
-    PROXY_PID=$!
-    py.test --proxy --spinnaker localhost WIDTH HEIGHT --bmp BMP_HOSTNAME
-    kill $PROXY_PID
-    
-    # On a machine on the same LAN as the spinnaker machine
-    spinnaker_proxy.py -stq SPINN_HOSTNAME
-
-We use this configuration to enable remote Travis-CI test runs to drive
-SpiNNaker hardware in Manchester.
-
-### Test coverage checking
-
-To get a test coverage report run one of the following:
-
-    # Summary (for rig module)
-    py.test --cov rig
-    
-    # List missed line numbers (for rig module)
-    py.test --cov rig --cov-report term-missing
-    
-    # Generate HTML report (for rig module)
-    py.test --cov rig --cov-report html
-
-
-### Code standards checking
-
-To test for coding standards problems run:
-
-    flake8 rig
-
-### Using Tox
-
-We also use [Tox](https://pypi.python.org/pypi/tox/1.8.1) to run tests against
-multiple versions of Python. To do this just execute `tox` in the root
-directory of the repository.
-
-
-### Building documentation
-
-Documentation is built by
-[Sphinx](http://sphinx-doc.org/)/[numpydoc](https://github.com/numpy/numpydoc).
-Dependencies can be installed using:
-
-    pip install -r requirements-docs.txt
-
-HTML documentation can be built using:
-
-    cd docs
-    make html
-
-HTML documentation is created in `docs/build/html/`.
-
-Note: `virtualenv` users using `--system-site-packages` and who have a system-wide
-installed version of sphinx may find that the build process fails. To install a
-local copy of Sphinx in the `virtualenv`, use:
-
-    pip install -I sphinx
+Overview
+--------
 
-The ReadTheDocs project page is available here: https://readthedocs.org/projects/rig/
+Rig does not attempt to enforce any particular application work flow but rather
+to provide a set of commonly useful utilities with well-defined, composable
+interfaces. Developers are encouraged to use whatever subset of these tools
```

Comment:
"Rig does not <mandate/require> any particular application workflow but rather provides a set of common utilities with well-defined, composable interfaces."

(I care more about the tense collision than the wording :wink:)